packages/xo-server/src/xo-mixins/rest-api.mjs

```diff
@@ -183,7 +183,10 @@ async function _getDashboardStats(app) {
   dashboard.nHosts = hosts.length
   dashboard.nHostsEol = nHostsEol
 
-  if (await app.hasFeatureAuthorization('LIST_MISSING_PATCHES')) {
+  const hasFeatureAuthorizationListMissingPatchesPromise = app.hasFeatureAuthorization('LIST_MISSING_PATCHES')
+  const remotesInfoPromise = app.getAllRemotesInfo()
+
+  if (await hasFeatureAuthorizationListMissingPatchesPromise) {
```

Comment:
Postponing the `await` in the same loop has no effect, do not complicate the code like that.